src/envs/behavior_options.py

```diff
@@ -25,6 +25,8 @@
         get_aabb_extent,
     )
     from igibson.robots.robot_base import BaseRobot  # pylint: disable=unused-import
+    from igibson.robots.behavior_robot import BRBody  # pylint: disable=unused-import
```

Comment:
what's the purpose of this import?

Reply:
woops - I forgot to use this. It's for the type-spec of the second argument of `get_delta_low_level_hand_action`